monitoring/monitorlib/kml.py

```diff
@@ -0,0 +1,101 @@
+# Module to parse KML file.
+
+from pykml import parser
+import re
+
+
+KML_NAMESPACE = {"kml":"http://www.opengis.net/kml/2.2"}
+
+def get_kml_root(kml_path):
+    kml = parser.parse(kml_path)
+    return kml.getroot()
+
+
+def get_folders(root):
+    return root.Document.Folder.Folder
+
+
+def get_polygon_speed(polygon_name):
+    """Returns speed unit within a polygon."""
+    result = re.search(r"\(([0-9.]+)\)", polygon_name)
+    return float(result.group(1)) if result else None
+
+def get_folder_details(folder_elem):
```

Comment:
This function is specific enough to rid_qualifier that it seems unlikely we would reuse it in another general `monitoring` tool.  So, we should probably put it in monitoring/rid_qualifier/kml.py rather than here (monitoring/monitorlib/kml.py presumably contains functions and tools that can be reused over many different monitoring tools).